doc/authentication-backends/HTTP-authentication-module.md

```diff
@@ -112,6 +112,15 @@ A body can be missing in the first data chunk read from a socket, leading to str
     * 200, `true` or `false` in body
     * anything else - will be treated as `false`
 
+### Method `get_certs`
+
+* **Description** Provides a list of certificates corresponding to a given user. Works like third party Certificate Authority service. The certificates must be given in `PEM` format.
+* **HTTP method** GET
+* **Type**: mandatory when verification with `ejabberd_auth_http` is used
+* **Return values**:
+  * `200`, when user exists, body consists of a list of certificates corresponding to user
```

Comment:
* `200`, when a user exists, the body consists of a list of certificates corresponding to them